pkg/pillar/devicenetwork/addrchange.go

```diff
@@ -175,7 +175,7 @@ func HandleAddressChange(ctx *DeviceNetworkContext) {
 			log.Functionf("HandleAddressChange: No change\n")
 		}
 	} else {
-		dnStatus = MakeDeviceNetworkStatus(log, *ctx.DevicePortConfig,
+		dnStatus = MakeDeviceNetworkStatus(log, ctx.Pending.PendDPC,
```

Comment:
Why did you change this? 
If it needs to change shouldn't the similar code in devicenetwork.go also change?
And it would make sense as a separate commit (in the same PR) to make it easier to follow the history and the motivation for the changes.

Reply:
Yeah, It should be a separate commit. Will remove this change from current diff.

Reply:
removed the unrelated change.